src/main/java/com/minecolonies/coremod/blocks/BlockBarrel.java

```diff
@@ -56,10 +57,10 @@ public BlockBarrel()
     }
 
     //todo: register block with new method
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)